packages/sdks/web-component/test/sdkScripts/grecaptcha.test.ts

```diff
@@ -0,0 +1,74 @@
+import { loadGRecaptcha } from '../../src/lib/descope-wc/sdkScripts/grecaptcha';
+
+describe('reCAPTCHA script', () => {
```

Comment:
mm this kinda conflicts ill need to move this to content which i wasn't planning but ill try